vendor-extra/JatsContentBundle/src/ViewConverter/ArticleTitleHeadingVisitor.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\ViewConverter;
+
+use FluentDOM\DOM\Element;
+use Libero\ViewsBundle\Views\SimplifiedVisitor;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverterVisitor;
+
+final class ArticleTitleHeadingVisitor implements ViewConverterVisitor
+{
+    use SimplifiedVisitor;
+
+    protected function doVisit(Element $object, View $view, array &$context = []) : View
+    {
+        return $view->withArgument('text', (string) $object);
```

Comment:
perhaps by decorating the `View` object, but as the population of Visitor grows there's the need for a standard way to log which `Visitor` has done what , to operate this correctly and understand what is clashing (if anything)

Reply:
Had thought the same, some mechanism to record this in the Symfony Profiler.